app/components/PillLabel.tsx

```diff
@@ -0,0 +1,30 @@
+import { tw } from '@helpwave/common/twind/index'
+
+export type PillLabelProps = {
+  count?: number,
+  state?: 'unscheduled' | 'inProgress' | 'done'
+}
+
+const PillLabel = ({ count = 0, state = 'unscheduled' }: PillLabelProps) => {
+  // case 'unscheduled' is default
+  let labelNumber = '1'
+  let text = 'unscheduled'
+  switch (state) {
+    case 'inProgress': labelNumber = '2'; text = 'in progress'; break
+    case 'done': labelNumber = '3'; text = 'done'; break
+    default: break
+  }
```

Comment:
use an map data structure instead
`switch` can contain any type of logic -a `map` is more restrictive

Reply:
The state can be an enum and should then be used in a map with the displayable text.